dsc_lib/src/configure/context.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use serde_json::Value;
+use std::collections::HashMap;
+
+pub struct Context {
```

Comment:
If params can potentially be used in direct resource invocations (in addition to configurations) then this file probably should be moved to a more shared place.

Reply:
There's no intent to have params supported for direct resource invocation, it's a feature of config only at this time